### ISSUE TITLE
Bug: Email address with + in username triggers invalid email format error.

### ISSUE THREAD

**Issue by hollajandro** [qdm12/ddns-updater#112]
**TLDR**: Email address with + on user side of address fails verification and triggers ERROR invalid email format error.

1. Is this urgent?

    - [ ] Yes
    - [x] No

2. What DNS service provider(s) are you using?

    - [x] Cloudflare
    - [ ] DDNSS.de
    - [ ] DonDominio
    - [ ] DNSOMatic
    - [ ] DNSPod
    - [ ] Dreamhost
    - [ ] DuckDNS
    - [ ] DynDNS
    - [ ] GoDaddy
    - [ ] Google
    - [ ] He.net
    - [ ] Infomaniak
    - [ ] Namecheap
    - [ ] NoIP

3. What's the version of the program?

    **See the line at the top of your logs**

    `Running version latest built on 2020-10-13T01:10:51Z (commit 860bc02)`

4. What are you using to run the container?

    - [ ] Docker run
    - [x] Docker Compose
    - [ ] Kubernetes
    - [ ] Docker stack
    - [ ] Docker swarm
    - [ ] Podman
    - [ ] Other:

5. Extra information

Logs:

Running version latest built on 2020-10-13T01:10:51Z (commit 860bc02)

🔧  Need help? https://github.com/qdm12/ddns-updater/issues/new
💻  Email? quentin.mcgaw@gmail.com
☕  Slack? Join from the Slack button on Github
💸  Help me? https://github.com/sponsors/qdm12
2020-10-20T21:26:33.038Z	ERROR	invalid email format

Configuration file (**remove your credentials!**):

{
    "settings": [
        {
            "zone_identifier": "",
            "provider": "cloudflare",                      
            "domain": "example.com",                                                   
            "host": "hostname",                                
            "ttl": 1,                                   
            "email": "user+name@gmail.com",  
            "key": "",
            "proxied": false,                            
            "ip_version": "ipv6"                              
        }                                                  
    ]                                                      
}                                                          

Host OS:

RancherOS 1.5.6

**Comment by qdm12** [comment]
Sorry for the delay; it should now be fixed if you re-pull the docker image

```sh
docker-compose pull
docker-compose up -d --force-recreate
```

Let me know if it still doesn't work. Thanks!
